lib/generators/my_zipcode_gem/templates/county_model.rb

```diff
@@ -1,5 +1,6 @@
+require 'memoist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.